docs/contributing/local-development.md

```diff
@@ -10,14 +11,33 @@
 
 Tilt offers a simple way of creating a local development environment.
 
-## Create a kind cluster
+## Create local k8s cluster
 
-Create a kind cluster with a local registry:
+```bash
+make deploy-dev
+```
+
+Open the `tilt` browser UI by pressing the "space" key when prompted. Note that `tilt` sets up a directory watcher and will automatically redeploy any code changes!
+
+You can check the "temporal-operator-controller-manager" Pod status with `kubectl get pods -n temporal-system -w`.
+
+## Apply Manifests
+
+Once the local cluster is created, start applying your desired manifests and let the temporal operator handle reconciliation:
 
 ```bash
-make dev-cluster
+# example
+make artifacts
+kubectl apply -f out/release/artifacts/temporal-operator.crds.yaml
+kubectl apply -f out/release/artifacts/temporal-operator.yaml
+kubectl apply -f examples/cluster-postgres/00-namespace.yaml
+kubectl apply -f examples/cluster-postgres/01-postgresql.yaml
+kubectl apply -f examples/cluster-postgres/02-temporal-cluster.yaml
+kubectl apply -f examples/cluster-postgres/03-temporal-namespace.yaml
 ```
 
+Note: if you wish to interact with the Temporal Web UI or frontend gRPC service, you should port forward the services to localhost:
```

Comment:
With the ending `:` we expect to get the command, but it's missing :)

Reply:
True. Actually I had put these commands in the last second
```bash
kubectl port-forward deployment/prod-ui 8080:8080 > /dev/null 2>&1 &
kubectl port-forward deployment/prod-frontend 7233:7233 > /dev/null 2>&1 &
```
but removed them before submitting PR because I didn't want to enforce the user to follow only one way of port forwarding.